src/main/protobuf/soc/playerState.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto2";
+
+package protos.soc;
+
+import "soc/inventory.proto";
+
+message PublicInventory {
+    required int32 numCards = 4;
+    required int32 numDevelopmentCards = 5;
+}
+
+message PerfectInventory {
+    required ResourceSet resourceCards = 1;
+    required DevelopmentCardSet playedDevCards = 2;
+    required DevelopmentCardSet canPlayDevCards = 3;
+    required DevelopmentCardSet cannotPlayDevCards = 4;
+}
+
+message ProbableInventory {
+    required ProbableResourceSet probableResourceCards = 1;
+    required DevelopmentCardSet playedDevCards = 2;
+    //TODO Add ProbableDevelopment cards
+}
+
+message PlayerState {
+    required string name = 1;
+    required int32 position = 2;
+    required int32 numPoints = 3;
+    required int32 numRoads = 4;
+    required int32 numKnights = 5;
+    required bool hasLongest = 6;
+    required bool hasLargest = 7;
+    required PublicInventory public = 8;
```

Comment:
Let's talk about the situation of of `Public` vs `Perfect` vs `Probable` it needs a phone call, I think.